app/helpers/value_helper.rb

```diff
@@ -2,6 +2,17 @@
 
 module ValueHelper
   # Renderers
+
+  # Given a fedora object, fetch the preferred object title
+  # @return [String] object title, as pulled from metadata, given the following order of preference:
+  #  1. MODS title, 2. identityMetadata label, 3. DC title, 4. Fedora label
+  def object_title(obj)
+    return obj.descMetadata.title_info.first.strip if obj.descMetadata.title_info.first.present?
```

Comment:
is this different than `obj.mods_title`?

Reply:
I'm sort of against adding all these conditions here as Ben is saying:

> Collection items should always have a MODS title, the other datastreams are there as backup in case something goes awry with creation of the descMetadata datastream.

This seems to indicate the other situations are exceptional.  Should we raise an error or log this (perhaps in Honeybadger.notify)?

Reply:
1. Didn't realize there was a method for that.  I looked into it and discovered that `obj.mods_title` is part of the `Editable` module in dor-services, which is only included in APO type objects.  Not sure why that is, but that method is not available to collections (or items for that matter).

2. Let me check with Ben on your idea for just throwing an exception when there is no title in MODs instead of all the conditionals.

Reply:
Ben liked this idea - implemented now and rebased/pushed.  Note that I decided to return the fedora label after notifying honeybadger so at least the app can continue for the user (with a potentially non-useful fedora label).  Asked Ben to confirm this strategy as well.